File: Client/src/app/services/model/championship.ts
import { Team } from './team';
import { Guid } from 'guid-typescript';

export interface Championship {

    id: Guid;
    name: string;
    isComplete?: boolean;
    anno: string;
    squadre?: Team[];
}

export class ChampionshipClass {
    constructor(
        public name: string,
        public anno: string,
    ) {}
}

File: 2.2/Data/Entities/Championship.cs

using System;
using System.Collections.Generic;

namespace fantacalcioApi.Data.Entities
{
    public class Championship
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public bool IsComplete { get; set; }
        public string Anno { get; set; }
        public List<Team> Squadre { get; set; }
    }

}
